src/startupscriptgenerator/node/scriptgenerator.go

```diff
@@ -215,6 +241,56 @@ func (gen *NodeStartupScriptGenerator) getPackageJsonMainCommand(packageJsonObj
 	return ""
 }
 
+func (gen *NodeStartupScriptGenerator) getProcessJsonCommand(userInputPath string) string {
+	processJsonPath := ""
+	if userInputPath != "" {
+		if strings.HasSuffix(userInputPath, ".json") {
+			processJsonPath = userInputPath
+		}
+	} else {
+		processJsonPath = "process.json"
+	}
+
+	if processJsonPath != "" {
+		processJsonFullPath := filepath.Join(gen.SourcePath, processJsonPath)
+		if common.FileExists(processJsonFullPath) {
+			debugCommand := ""
+			if gen.isDebugging() {
+				debugCommand = gen.getNodeWrapperDebugCommand()
+			}
+			return debugCommand + getPm2StartCommand(processJsonPath)
```

Comment:
Do you want to pass `processJsonFullPath` instead?

Reply:
No, I wanted to keep the relative path since we're already in the app's directory.